test/officerManager.js

```diff
@@ -0,0 +1,336 @@
+/* global describe it before after */
+// During the test the env variable is set to test
+process.env.NODE_ENV = 'test'
+const OfficerManager = require('../api/models/OfficerManager')
+const User = require('../api/models/User')
+// Require the dev-dependencies
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const server = require('../server')
+let serverInstance = null
+let app = null
+const expect = chai.expect
+// tools for testing
+const tools = require('../util/testing-utils/tools.js')
+
+chai.should()
+chai.use(chaiHttp)
+
+function initializeServer () {
+  serverInstance = new server.Server()
+  serverInstance.openConnection()
+  app = serverInstance.getServerInstance()
+}
+
+function terminateServer (done) {
+  serverInstance.closeConnection(done)
+}
+
+// Our parent block
+describe('OfficerManager', () => {
+  before(done => {
+    initializeServer(app)
+
+    // Before each test we empty the database
+    tools.emptySchema(OfficerManager)
+    tools.emptySchema(User)
+    done()
+  })
+  after(done => {
+    terminateServer(done)
+  })
+
+  let token = ''
+
+  describe('/POST submit', () => {
```

Comment:
for every API that requires a token, please also test that by submitting a form without a token will return `UNAUTHORIZED`.

Reply:
no token require for submit